mingw-w64-cubelib/PKGBUILD

```
@@ -4,27 +4,34 @@
 _realname=cubelib
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=4.8
+pkgver=4.8.2
 pkgrel=1
 pkgdesc="Library for reading and writing a multi-dimensional performance space (mingw-w64)"
 arch=('any')
-mingw_arch=('mingw32' 'mingw64' 'ucrt64')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clangarm64')
 url="http://www.scalasca.org/software/cube-4.x/download.html"
 license=('BSD')
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
          "${MINGW_PACKAGE_PREFIX}-zlib"
          "${MINGW_PACKAGE_PREFIX}-qt5-base")
-makedepends=("${MINGW_PACKAGE_PREFIX}-cc") 
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
+             $([[ ${MINGW_PACKAGE_PREFIX} != *-clang-* ]] || echo "${MINGW_PACKAGE_PREFIX}-flang"))
```

```suggestion
             $([[ ${MSYSTEM} == CLANG32 ]] || echo "${MINGW_PACKAGE_PREFIX}-fc"))
```
Is there a reason why not using gfortran on GCC environments?

---

Hum.. I didn't want to touch the existing behavior for the MINGW* envs. I was just trying to add CLANG64 and CLANGARM64. Should I add gfortran as a dep for the MINGW* envs?

---

just `${MINGW_PACKAGE_PREFIX}-fc` which is gfortran on GCC envs.